.github/test-categories.yaml

```diff
@@ -24,12 +24,12 @@ cloud:
   - elasticsearch-rest
 core:
   - core
```

Comment:
Does it make sense to have a separate core test category with a single test module?

Reply:
I will revisit it after the work on the bootstrap as there could be some new tests belonging to the core category, this is primary aimed to have a clear baseline for the upcoming work